alembic/versions/01182f408d4c_initial_migration.py

```diff
@@ -7,9 +7,10 @@
 
 """
 import sqlalchemy as sa
-from alembic import op
 from sqlalchemy.dialects import postgresql
 
+from alembic import op
```

Comment:
This is also incorrect, for the same reason as above. I'm not going to comment on the other alembic moves.